setup.sh

```diff
@@ -11,18 +11,25 @@ set -e
 
 # Initialize project dependency directories.
 init(){
+  ENVVAR_SAMPLE=.env_SAMPLE
+  ENVVAR=.env
```

Comment:
Won't this overwrite our `.env` files every time we have to re-run `setup.sh`, like when switching to a branch w/ new deps?